pipeline/07-export.R

```diff
@@ -803,6 +854,17 @@ for (town in unique(assessment_pin_prepped$township_code)) {
   rm(wb)
 }
 
+### NOTE ###
+# OpenXLSX is not perfect and messes up the macros and formatting on saved
+# workbooks. To finish each workbook, you must manually:
+
+# 1. Open the Neighborhood Breakouts sheet and ensure that the values are
+#    all formatted correctly in the pivot table; if not (e.g. if
+#    `Average of YoY ∆ %` is formatted as a date when it should be a percentage)
+#    then manually update the formatting by selecting
+#    PivotTable Fields > Values > {fieldname} > Value Field Settings... >
+#    Number Format.
```

Comment:
It's kind of a bummer that we have to do this manual editing, but I tried at least to make the instructions match the format used for the `api` stage so that at least we have some consistency between stages:

https://github.com/ccao-data/model-res-avm/blob/2a86b059ac3bcf77583b55790f216494084947af/pipeline/08-api.R#L177-L186